### PR TITLE
Adds possibility to set node stereotype

### PR DESCRIPTION
**Pull request by schmetzyannick**
**Thanks for contributing to the Docker-Selenium project!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines, applied for this repository.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

See Issue #1665 

Seems to Work:
image

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -779,6 +779,9 @@ In both cases, it is necessary to tell the Node where the Event Bus is, so it ca
 the purpose of the `SE_EVENT_BUS_HOST`, `SE_EVENT_BUS_PUBLISH_PORT` and `SE_EVENT_BUS_SUBSCRIBE_PORT` environment
 variables.
 
+In some cases, for example if you want to tag a node, it might be necessary to supply a custom stereotype to the node config. The environement variable `SE_NODE_STEREOTYPE`
```
Comment (diemol) [inline README.md:782]: Can you please add an example of this?

Comment (schmetzyannick) [reply README.md:782]: For the docker command below?

Comment (diemol) [reply README.md:782]: Yes, how one would start a container and use the `SE_NODE_STEREOTYPE` env var.

Comment (schmetzyannick) [reply README.md:782]: Done

**Review by diemol — APPROVED** [review]
Thank you, @schmetzyannick!

**Pull request merged**